wweval/R/ms_fig4.R

```diff
@@ -636,6 +650,9 @@ FGGG
     legend.justification = "left"
   ) #+ plot_annotation(tag_levels = "A") #nolint not working
   fig4
+
+  cfaforecastrenewalww::create_dir(fig_file_dir)
```

Comment:
since we're planning to move away from `cfaforecastrenewalww` wouldn't it be better to use base `dir.create` or `fs::dir_create` here if at all possible? The only difference is the directory here. Is that still necessary in the context in which you're running things here:

https://github.com/CDCgov/wastewater-informed-covid-forecasting/blob/8f79e70889c4ca8a1881d84118b3635c6282776e/cfaforecastrenewalww/R/utils.R#L127-L132

Reply:
I will try out using `dir.create()`

Reply:
Yeah I'll go ahead and replace with this throughout